.github/workflows/windows-docker.yaml

```diff
@@ -0,0 +1,57 @@
+#
```

Comment:
What do the errors below mean?

Reply:
I don't know, but they are annoying. They've always been there when a build fails....

Reply:
Thanks. Was just wondering.